docs/report/report.tex

```diff
@@ -189,174 +189,164 @@ \subsection{Deployment Cycle}
-
+Die ersten Probleme waren die Auswahl des Technologie-Stacks sowie die Auswahl der benötigten Hardware.
+Auch musste sich das Team das Projekt so entsprechend planen, dass jedes Teammitglied durchgehend sinnvoll arbeiten kann.
 \subsubsection{Lösungen}
-\colorbox{red}{TODO:}Welche Lösungen haben wir gefunden?
-
+Da bereits einige Teammitglieder Erfahrung mit verschiedenen Technologie-Stacks und ähnlichen Problemen hatten, wurde sich auf den Technologie-Stack geeinigt, bei dem die meiste Erfahrung im Team bestand.
+Dieser Bestand aus den oben beschriebenen Technologien.
+Um ein sinnvolles kontinuierliches Arbeiten jedes Teammitglieds zu gewährleisten, einigte sich das Team darauf ein ständig gefülltes Ticket-Pool im Repository zu führen, sodass jedes Teammitglied jederzeit selbstständig eine Aufgabe finden und bearbeiten konnte.
 \subsubsection{Product Increment}
-\colorbox{red}{TODO:}Was ist am Ende dabei rumgekommen?
-
+Insgesamt kam am Ende des Milestones nicht im klassischen Sinne ein Product Increment zustande, vielmehr wurden die Grundbausteine für die Entwicklung und das Teamwork gelegt.
+Es wurde sich auf das bereits erwähnte Technologie-Stack entschieden sowie als Hardware ein Raspberry Pi gewählt.
+Für die Entwicklung wurde ein Mono-Repository mit lerna\footnote{https://lerna.js.org/} aufgesetzt, welches Front- und Backend zusammenfasste.
+Durch die Open Source Natur dieses Projektes wurde eine verständliche und hilfreiche Dokumentation für das Projekt  nötig.
+So wurde auch das Repository mit einer einleitenden Readme, einem \glqqHow to Contribute\grqq Guide und einem gefüllten Backlog versehen.
 \subsubsection{Retrospektive}
-\colorbox{red}{TODO:}Was haben wir dabei gelernt? Neue Erkentnisse? Neue Sichtweisen?
-Was lief gut, neu gelernt, was lief nicht so gut, was verbessern?
-\begin{comment}{
-
-Lief Gut
-- Zusammenarbeit und Team Environment
-- Meetings und Zusammenarbeit
-- Github Workflow
-- Anton!!!
-- Richtig viel gelernt.
-- Sau viel Zeit Einsatz von Anton, muss man mal wertschätzen.
-- gruppenklima
-- Die Kommunikation war klasse, genauso Teamwork.
-
-Neu Gelernt
-- Vue
-- Feathers
-- Tailwind
-- CI
-- undefined & null
-- feathers
-- vue
-- Code / PR Review
-
-Nicht so gut
-- globale Probleme waren meh
-- bei Problemen (nicht weitergekommen) nicht sofort bescheid sagen
-- Wörtlicher Kundenwunsch wurde viel zu spät in dem sprint behandelt. Auch wenn hello wolrd cool war müssen wir vielleicht in dem nächsten sprint härter priorisieren.
-
-Verbessern
-- Aufgabenaufteilung
-- Code reviews häufiger machen
-- Aufgaben früher
-- Teils lange Wartezeiten
-- mehr den aktuellen stand kommunizieren
-- Pünktlichkeit und rumorganisation der stand ups.
-}
+Nach dem ersten Meilenstein fand noch keine nennenswerte Retrospektive statt.
 
 \subsection{Meilenstein 2}
-\colorbox{red}{TODO:}Kurze Einleitung,  von wann bis wann ging}
+Der zweite Meilenstein wurde für den Zeitraum 11.11.2020 bis 25.11.2020 angelegt.
```

Comment:
```suggestion
Der zweite Meilenstein wurde für den Zeitraum vom 11.11.2020 bis zum 25.11.2020 angesetzt.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/409/commits/db7fedecbe3f12f759a2c504c4b9f0e5a7061fee